tests/stub/session_notification_filters/test_driver_notification_filters.py

```diff
@@ -0,0 +1,52 @@
+from nutkit.frontend import Driver
+import nutkit.protocol as types
+from tests.shared import TestkitTestCase
+from tests.stub.session_notification_filters.notification_filters_base import (
+    NotificationFiltersBase,
+)
+from tests.stub.shared import StubServer
+
+
+class TestDriverNotificationFilters(TestkitTestCase, NotificationFiltersBase):
+    required_features = (types.Feature.BOLT_5_1,
+                         types.Feature.API_DRIVER_NOTIFICATION_FILTERS)
+    _auth = types.AuthorizationToken("basic", principal="neo4j",
+                                     credentials="pass")
+
+    def setUp(self):
+        super().setUp()
+        self._server = StubServer(9010)
+        self._uri = "bolt://%s" % self._server.address
+        self._driver = None
+
+    def tearDown(self):
+        self._server.reset()
+
+        if self._driver:
+            self._driver.close()
+        return super().tearDown()
+
+    def _new_driver(self, notifications):
+        return Driver(self._backend, self._uri, self._auth,
+                      notification_filters=notifications)
+
+    def _run_test_get_summary(self, filters, script_params,
+                              script_name="driver_notification_filters.script"
+                              ):
```

Comment:
```suggestion
    def _run_test_get_summary(
        self, filters, script_params, 
        script_name="driver_notification_filters.script"
    ):
```
Either way, we'll be left with a very sad line of code
> `):`

Reply:
just renamed it script, so it it's a bit better.